File: NODE_JS/Tuần 3/Bai2.js
a = 123;
b = 9;

max_index = Math.ceil(a/b);

console.log(max_index);

var index = 12

if(max_index < index){

    index = max_index;
}

c = b * index;
if(c > a){
    c = a;
}
for(let i = c - b -1; i < c; i++){
    console.log(i);
}

File: PYTHON/Tuần 7/Bai2.py
import math
a = [2,4,6,8,10,12,14,16,18,20]
b = []
for i in range (0,10,1):
    b.append(a[i]*a[i])
print(b)

c = [2,-3,5,7,8,-10]
d = []
for i in range (0,len(c),1):
    if c[i] < 0:
        d.append(c[i] * c[i])
    else:
        d.append(c[i])

print(d)